src/enemyLocations/disneyCastle.ts

```diff
@@ -9,7 +9,7 @@ export const disneyCastleEnemyLocations: EnemyLocation[] = [
 		description: "Escort Queen Minnie Part I",
 		world: "0C",
 		room: "02",
-		event: "02",
+		eventGroup: ["02","02","01"],
```

Comment:
Please revert the changes in this file as I haven't ported the enemy rando improvements made by thundrio to this branch yet, I will accept them in a different PR once I do that.